chainer/variable.py

```diff
@@ -948,6 +963,12 @@ def grad(self):
         and error.
 
         """
+        if not self._grad_valid:
+            raise RuntimeError(
+                'Cannot retrieve Variable.grad because it\'s not valid. '
```

Comment:
```suggestion
                'Cannot retrieve Variable.grad because it must be set '
```

Reply:
How about "... because neither it's set beforehand nor Variable.cleargrad() is called" ?

Reply:
`Neither... nor...` is difficult to understand English. I think my first revision is more clear.

Reply:
Hmm.
I feel your version sounds to suggest the cause is "it must be ..." and does not state whether "it actually is".

Reply:
How about removing "because it's not valid" from the original version, leaving two sentences?

Reply:
So,
```Cannot retrieve Variable.grad. Either it must be set beforehand or Variable.cleargrad() must be called.```

I'm ok with that.

Reply:
Actually, for more clarity, should be,

```Cannot retrieve Variable.grad. Either it must be set manually or Variable.cleargrad() must be called beforehand.```

Reply:
Thank you for correction. Fixed as such.